server/util/vueSsrCache.js

```diff
@@ -28,20 +26,12 @@ module.exports = function wrapper(cache) {
 			});
 		},
 		set(key, value) {
-			cache.set(key, value.html, 1 * 60 * 60, (error, success) => {
+			cache.set(key, value.html, { expires: 0 }, (error, success) => {
```

Comment:
Because the release hash is included in the component cache keys, it's ok for them to never expire (this is the same behavior as what is already happening, just explicit about it instead)